src/globals.d.ts

```diff
@@ -3,26 +3,31 @@
 import { WebpackChunkGlobal } from "./types/discord";
 import * as replugged from "./renderer/replugged";
 import { RepluggedNativeType } from "./preload";
-import type React from "react";
+import type Lodash from "lodash";
 
 declare global {
   export var appSettings: {
     set(setting: string, v: unknown): void;
   };
 
   interface Window {
-    React: typeof React;
     RepluggedNative: RepluggedNativeType;
+    DiscordNative: typeof DiscordNative;
     replugged: typeof replugged;
     // eslint-disable-next-line @typescript-eslint/naming-convention
     webpackChunkdiscord_app: WebpackChunkGlobal;
+    _: typeof _;
```

Comment:
Shouldn't this be in common modules

Reply:
It is, but lodash is available globally and I see no harm in allowing devs to use that instead of an inconvenient import.

Reply:
I think we should do the same with React then. That way ppl can import it directly.

Reply:
Hmm, maybe, although `window._` is created by Discord while `window.React` would be us polluting the window...out of scope for this though

Reply:
I think it's worth it but we can create a separate PR for that.